documentation/src/main/asciidoc/performance_guide/performance_guide.asciidoc

```diff
@@ -0,0 +1,232 @@
+= Infinispan {infinispanversion} Performance Guide
+The Infinispan community
+:toc2:
+:icons: font
+:toclevels: 3
+:numbered:
+:stem:
+
+This guide will give you information and tweaks about tuning Infinispan performance (both server and library mode).
+
+== Capacity planning
+
+Infinispan stores data in memory. Depending on operating mode data is stored in serialized form (server mode) or directly
+directly in memory (library mode). Data size can be estimated using sophisticated tools like  http://openjdk.java.net/projects/code-tools/jol/[Java Object Layout]
```

Comment:
Also you provide a way to estimate the size of a POJO using JOL, but this doesn't provide information about serialized size, which depends on the marshaller. It would be nice to have a similar tool